README.md

```diff
@@ -106,3 +106,18 @@ Beyond the officially supported [flow environments](https://docs.signac.io/proje
 
 * Fry
 * Kestrel
+
+### .sh file instructions
+<details>
+     <summary>Clean.sh (Click to expand):</summary>
+     Clean.sh is a shell script file that will remove the workspace folder, including the files inside of it, within the present planckton-flow clone you are using. To run this shell script:
+    ```bash
+    bash clean.sh 
+     ```
+     in your command line.**MAKE SURE YOU ARE IN THE CLONE OF PLANCKTON_FLOW THAT YOU WANT THE WORKSPACE REMOVED FROM.
+     <summary>Print_status.sh (Click to expand):</summary>
+     Print_status.sh is a shell script file that prints out the status of your recently submitted jobs. To run this shell script:
+     ```bash
+     bash print_status.sh
+      ```
```

Comment:
```suggestion

     ```bash
     bash print_status.sh
     ```

```

Reply:
Awesome, thanks for the feedback and additions!